build.gradle.kts

```diff
@@ -39,6 +41,17 @@ allprojects {
 }
 
 subprojects {
+  apply(plugin = "com.diffplug.spotless")
+  configure<SpotlessExtension> {
+    setEnforceCheck(false)
```

Comment:
So that CI doesn't break when formatting is off.

Reply:
Thanks.